packages/unmock-core/src/generator.ts

```diff
@@ -52,8 +52,15 @@ import {
   Schema,
 } from "./service/interfaces";
 import { ServiceStore } from "./service/serviceStore";
-export const seedHack = {
+export const runnerHack = {
   seed: 0,
+  optionalsProbability: 1.0,
+  minItems: 0,
+};
+export const resetRunnerHack = () => {
```

Comment:
Something like `runner.configuration = DEFAULT_CONFIGURATION` would maybe reduce the need to remember to define these in two places.

Reply:
It's actually meant to be used within the generator, to reset `minItems`.
I can't decide if it's better after generation or after `runner` is complete. But TBH I don't understand yet how `runner` works, so I preferred putting it here :)
(see line num 727)

Reply:
Now that I do, I'll just move it there.